x-pack/legacy/plugins/security/server/lib/authentication/providers/kerberos.ts

```diff
@@ -109,32 +88,18 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
    * @param request Request instance.
    * @param state State value previously stored by the provider.
    */
-  public async deauthenticate(request: Legacy.Request, state?: ProviderState) {
+  public async deauthenticate(request: Legacy.Request, state?: ProviderState | null) {
     this.debug(`Trying to deauthenticate user via ${request.url.path}.`);
 
-    if (!state || !state.accessToken) {
+    if (!state) {
```

Comment:
__note:__ if we have a state that doesn't have an `accessToken` in it - it's a bug, and I'd rather notice it earlier.